index.html

```diff
@@ -988,6 +988,10 @@
         </p>
         <p>Special thanks to those that helped pioneer the first edition of this model, including especially: Charu Chandiram, Peter Freiling, Nathan Furtwangler, Thomas Olsen, Matt Rakow, Ramu Ramanathan, Justin Rogers, Jacob Rossi, Reed Townsend and Steve Wright.</p>
     </section>
+    <section class="appendix">
+        <h1>Security and Privacy Considerations</h1>
+        <p>There are no known security or privacy impacts resulting from the features outlined in this specification.</p>
```

Comment:
I'll note again that without any good examples out there, it's difficult for a layperson like me to even understand what this security/privacy section should look like. If the expectation is to have something along the lines you mentioned there, with detailed bullet points, then it would be ace to see that as an actual example - it may be clear once reading the whole of https://www.w3.org/TR/security-privacy-questionnaire/, but on skim-reading it it wasn't apparent ;)